From 476d87ae8e503d3b5a51b081452dcf92773b2c27 Mon Sep 17 00:00:00 2001
From: Alexander Graf <agraf@suse.de>
Date: Fri, 26 May 2017 18:12:22 +0200
Subject: [PATCH 3/3] dpaa2: Remove ro_after_init usage
Patch-mainline: Never, only needed to make code work with SP3 kernel
References: bsc#1041022

SP3 does not know ro_after_init yet. Just remove the flag from the struct
definition.

Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c b/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c
index 49127ac..9091ea5 100644
--- a/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c
+++ b/drivers/staging/fsl-mc/bus/irq-gic-v3-its-fsl-mc-msi.c
@@ -52,7 +52,7 @@ static int its_fsl_mc_msi_prepare(struct irq_domain *msi_domain,
 	return msi_info->ops->msi_prepare(msi_domain->parent, dev, nvec, info);
 }
 
-static struct msi_domain_ops its_fsl_mc_msi_ops __ro_after_init = {
+static struct msi_domain_ops its_fsl_mc_msi_ops = {
 	.msi_prepare = its_fsl_mc_msi_prepare,
 };
 
-- 
1.8.5.6

